vocabularies/UI.xml

```diff
@@ -1389,6 +1390,59 @@ The trend is
         </Member>
       </EnumType>
 
+      <Term Name="Note" Type="UI.NoteType" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Visualization of a note attached to an entity" />
+        <Annotation Term="Core.LongDescription">
+          <String>Administrative data is given by the annotations
+          [`Common.CreatedBy`](Common.md#CreatedBy),
+          [`Common.CreatedAt`](Common.md#CreatedAt),
+          [`Common.ChangedBy`](Common.md#ChangedBy),
+          [`Common.ChangedAt`](Common.md#ChangedAt)
+          on the same entity type.</String>
+        </Annotation>
+      </Term>
+      <ComplexType Name="NoteType">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="Title" Type="Edm.String" Nullable="true">
+          <Annotation Term="Core.Description" String="Title of the note" />
+          <Annotation Term="Core.LongDescription">
+            <String>The title of a note is hidden with an annotation `@UI.Note/Title/@UI.Hidden`, not with
+            an annotation on the property targeted by `@UI.Note/Title`.</String>
+          </Annotation>
+          <Annotation Term="Validation.ApplicableTerms">
+            <Collection>
+              <String>UI.Hidden</String>
+            </Collection>
+          </Annotation>
+        </Property>
+        <Property Name="Content" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Content of the note, as a string" />
+          <Annotation Term="Core.LongDescription">
+            <String>The property targeted by `@UI.Note/Content` must be annotated with `Core.MediaType` and
+            `Common.SAPObjectNodeTypeReference`. When it is tagged with `Core.IsLanguageDependent`,
+            another property of the same entity type that is tagged with [`Common.IsLanguageIdentifier`](Common.md#IsLanguageIdentifier)
+            determines the language of the note.</String>
+          </Annotation>
+          <Annotation Term="Validation.ApplicableTerms">
+            <Collection>
+              <String>Core.MediaType</String>
+              <String>Core.IsLanguageDependent</String>
+              <String>Common.SAPObjectNodeTypeReference</String>
+            </Collection>
+          </Annotation>
+        </Property>
+        <Property Name="Type" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="A type used for grouping notes" />
```

Comment:
I still cannot grap this - who defines these types (assuming a CAP application wants to use notes)? And why is it nullable: false - why should I always want to group my notes.

Reply:
@thomas-kaltenbach, please answer.

Reply:
> I still cannot grap this - who defines these types (assuming a CAP application wants to use notes)? And why is it nullable: false - why should I always want to group my notes.

In the followed comment you can see a screenshot of the notes component https://github.com/SAP/odata-vocabularies/pull/221#issuecomment-1412315655.
A Note Type is defines a group of notes for example "Product Description" or "Comments". The application can reuse an existing one or define a new one. So far, applications and UX have required us to display them by default in groups. Which also makes sense from my point of view.
In addition you can define at the note type the content-type, if a title should be displayed and so on.

Reply:
Thanks, @thomas-kaltenbach! The property "Type defines a group, but if an app only has a plain list of notes, this property doesn't really matter, right. If so, we shouldn't define it as Nullable=false.

Reply:
The "type" also defines how the group should be displayed. We don't have a default behavior how to handle the UI when no configuration is available. From our side, the note type and the configuration attached to it are required.
Also, the note type is persisted as a foreignkey at the note content in the database because we are a reuse component and need it for the where/on condition. If we say that this could be nullable, we would be making the behaviour very complex. To be honest, I don't even know if that fits into the whole concept.
if there are any further doubts, we can also discuss it in a meeting.